docs/development/cherry-picks.md

```diff
@@ -0,0 +1,44 @@
+# Overview
+
+This document explains how to initiate cherry picks on release branches within the `gardener/gardener` repository.
+
+- [Prerequisites](#prerequisites)
+- [Initiate a Cherry Pick](#initiate-a-cherry-pick)
+
+## Prerequisites
+
+Before you initiate a cherry pick, make sure that the following prerequisites are accomplished.
+
+- A pull request merged against the `master` branch.
+- The release branch exists (example: [`release-v1.18`](https://github.com/gardener/gardener/tree/release-v1.18))
+- Have the `gardener/gardener` repository cloned as follows:
+  - the `origin` remote points to your fork
+  - the `upstream` remote points to the Gardener github org
```

Comment:
What do you think about to mention that these values can be overwritten with `FORK_REMOTE`  and `UPSTREAM_REMOTE` environment variables respectively?

Reply:
Nice suggestion, I now added this detail!